pyproject.toml

```diff
@@ -1478,7 +1478,6 @@ module = [
     "tests.sentry.db.models.fields.bitfield.test_bitfield",
     "tests.sentry.db.models.fields.test_jsonfield",
     "tests.sentry.db.models.fields.test_picklefield",
-    "tests.sentry.db.models.test_utils",
```

Comment:
did you mean to remove this?

Reply:
these seem to happen randomly from time to time -- the file is updated automatically by CI, i wouldn't worry about it.